e2e/vm/support_bundle_test.go

```diff
@@ -0,0 +1,159 @@
+	ginkgo.Describe("Support bundles", func() {
+		ginkgo.It("Should generate a support bundle", func() {
+			command.Run(o, "support-bundle", "generate")
+			entries, err := os.ReadDir(".")
+			gomega.Expect(err).Should(gomega.BeNil())
+			bundleExists := false
+			for _, dirEntry := range entries {
+				if strings.Contains(dirEntry.Name(), "finch-support") {
+					_, err := os.Stat(dirEntry.Name())
+					if err == nil {
+						bundleExists = true
+					}
+
+					err = os.Remove(dirEntry.Name())
+					gomega.Expect(err).Should(gomega.BeNil())
+				}
+			}
+			gomega.Expect(bundleExists).Should(gomega.BeTrue())
+		})
+		ginkgo.It("Should generate a support bundle with an extra file included", func() {
+			includeFilename := fmt.Sprintf("tempTestfile%s", time.Now().Format("20060102150405"))
+			_, err := os.Create(includeFilename)
+			gomega.Expect(err).Should(gomega.BeNil())
+			defer func() {
+				err := os.Remove(includeFilename)
+				gomega.Expect(err).Should(gomega.BeNil())
+			}()
+
+			command.Run(o, "support-bundle", "generate", "--include", includeFilename)
+			entries, err := os.ReadDir(".")
+			gomega.Expect(err).Should(gomega.BeNil())
+			bundleExists := false
+			for _, dirEntry := range entries {
+				if strings.Contains(dirEntry.Name(), "finch-support") {
+					_, err := os.Stat(dirEntry.Name())
+					if err == nil {
+						bundleExists = true
+					}
+
+					reader, err := zip.OpenReader(dirEntry.Name())
+					gomega.Expect(err).Should(gomega.BeNil())
+
+					zipBaseName := path.Base(dirEntry.Name())
+					zipPrefix := strings.TrimSuffix(zipBaseName, path.Ext(zipBaseName))
+					_, err = reader.Open(path.Join(zipPrefix, "misc", includeFilename))
+					gomega.Expect(err).Should(gomega.BeNil())
+
+					err = os.Remove(dirEntry.Name())
+					gomega.Expect(err).Should(gomega.BeNil())
+				}
+			}
+			gomega.Expect(bundleExists).Should(gomega.BeTrue())
+		})
+		ginkgo.It("Should generate a support bundle with a default file excluded", func() {
+			command.Run(o, "support-bundle", "generate", "--exclude", "serial.log")
+			entries, err := os.ReadDir(".")
+			gomega.Expect(err).Should(gomega.BeNil())
+			bundleExists := false
+			for _, dirEntry := range entries {
+				if strings.Contains(dirEntry.Name(), "finch-support") {
+					_, err := os.Stat(dirEntry.Name())
+					if err == nil {
+						bundleExists = true
+					}
+
+					reader, err := zip.OpenReader(dirEntry.Name())
+					gomega.Expect(err).Should(gomega.BeNil())
+
+					zipBaseName := path.Base(dirEntry.Name())
+					zipPrefix := strings.TrimSuffix(zipBaseName, path.Ext(zipBaseName))
+					_, err = reader.Open(path.Join(zipPrefix, "logs", "serial.log"))
+					gomega.Expect(err).ShouldNot(gomega.BeNil())
+
+					err = os.Remove(dirEntry.Name())
+					gomega.Expect(err).Should(gomega.BeNil())
+				}
+			}
+			gomega.Expect(bundleExists).Should(gomega.BeTrue())
+		})
+		ginkgo.It("Should generate a support bundle with the same file included and excluded", func() {
```

Comment:
Something like "Should generate a support bundle with the file excluded when it is both in --included and --exclude". Including both of the result and condition in the description can make the test log more readable.

Reply:
Same for other descriptions.